<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Add Record</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_age">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Job</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_job">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_add_record">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>231</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_view_all">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_view_names">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>180</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View names</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Remove Record</string>
    </property>
    <widget class="QPushButton" name="pushButton_remove">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_index">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Index</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clean">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>clean</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>275</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
